src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.tsx

```diff
@@ -119,6 +119,11 @@ const ConditionsCallout: React.FC<IConditionsCalloutProps> = (props) => {
                     payload: { values: newValues }
                 });
             } else {
+                // If user types infinity or -infinity handle that case (JSON parsing turns numerical infinities to null)
+                if (newValues[0] === Infinity || newValues[0] === -Infinity) {
```

Comment:
isn't newValues [min, max] tuple? shouldn't we check index 1 as well and do casting for Infinity if it is?